Assets/HoloToolkit/Utilities/Editor/ProjectSettingsWindow.cs

```diff
@@ -0,0 +1,274 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.RegularExpressions;
+using UnityEditor;
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+	/// <summary>
+	/// Renders the UI and handles update logic for HoloToolkit/Configure/Apply HoloLens Project Settings.
+	/// </summary>
+	public class ProjectSettingsWindow : AutoConfigureWindow<ProjectSettingsWindow.ProjectSetting>
+	{
+		// Nested Types
+		public enum ProjectSetting
+		{
+			ActiveBuildToWsa,
```

Comment:
[.Net Capitalization Conventions](https://msdn.microsoft.com/en-us/library/ms229043(v=vs.110).aspx) dictate that the PascalCasing convention be used for all identifiers except parameter names. A special case is only made for two-letter acronyms (not three letter acronyms).

Although Unity does upper case three letter acronyms and Microsoft does upper case three letter acronyms in C++ code, Microsoft does **NOT** upper case three letter acronyms in UWP code. Feel free to search the [Windows API reference for Windows Runtime apps](https://msdn.microsoft.com/en-us/library/windows/apps/br211377.aspx) and you will find that no three letter acronyms (including UWP and WSA) are capitalized.